### DISCUSSION TITLE
Kafka-UI connection count and other client metrics

### DISCUSSION THREAD

**Discussion by odoszhanov** [provectus/kafka-ui#4395] [General]
Hey! Is there a way to check the connections count coming from Kafka-UI? We might needed these metrics to have better observability from client side. Imagine if you're using MSK-Serverless which do not provide cluster level metrics on connections count (but do have a quota limit for it), additional client metrics might be useful to see if you're approaching limits.

**Comment by Haarolean** [comment]
There are messages polling metrics available: https://github.com/provectus/kafka-ui/pull/4069, that's perhaps all there is
